pkg/classroom/http.go

```diff
@@ -33,9 +33,29 @@ func ListClassrooms(client api.RESTClient, page int, perPage int) ([]Classroom,
 	return response, nil
 }
 
-func ListAcceptedAssignments(client api.RESTClient, assignmentID int, page int, perPage int) (AcceptedAssignmentList, error) {
+func getAssignmentList(client api.RESTClient, assignmentID int, page int, perPage int) ([]AcceptedAssignment, error) {
 	var response []AcceptedAssignment
 	err := client.Get(fmt.Sprintf("assignments/%v/accepted_assignments?page=%v&per_page=%v", assignmentID, page, perPage), &response)
+	return response, err
+}
+
+func ListAcceptedAssignments(client api.RESTClient, assignmentID int, page int, perPage int) (AcceptedAssignmentList, error) {
```

Comment:
`pkg/classroom/http.go` is only meant to contain API calls.

Lets move `ListAcceptedAssignments` and `ListAllAcceptedAssignments` to `cmd/gh-classroom/shared/shared.go`.

This way it can also be used in the [`accepted` package. ](https://github.com/github/gh-classroom/blob/main/cmd/gh-classroom/accepted-assignments/accepted_assignments.go)

Reply:
Done! Reference commit 4f532156311e466dde9ddba2db047b76494e5fd6